bootstrap/js/contact.js

```diff
@@ -0,0 +1,76 @@
+//Add an event listener for the form submission to create a new user
+
+const form = document.querySelector("form");
+
+
+const formEvent = form.addEventListener('submit', event => {
+   event.preventDefault(); 
+   const username = document.querySelector("#username").value;
+   const email = document.querySelector("#email").value;
+   const message = document.querySelector("#message").value;
+
+   const user = { username, email, message };
+
+
+   createUser(user);
```

Comment:
You're calling it here though. Which is the right place. So I think you should remove it from line 34.

Reply:
thanks, I will do that